-D__GLIBC__=2 is required to build on *-musl,
it's normally defined by features.h of glibc.
--- Makefile.orig
+++ Makefile
@@ -11,7 +11,7 @@
 DESTDIR:=
 
 COPT_FLAGS:=-O2
-CFLAGS:=$(COPT_FLAGS) -Wall -Wunused -I$(KERNEL_DIR)/include/ -Iinclude/ -DARPTABLES_VERSION=\"$(ARPTABLES_VERSION)\" #-g -DDEBUG #-pg # -DARPTC_DEBUG
+override CFLAGS+=-Wall -D__GLIBC__=2 -Iinclude/ -DARPTABLES_VERSION=\"$(ARPTABLES_VERSION)\"
 
 ifndef ARPT_LIBDIR
 ARPT_LIBDIR:=$(LIBDIR)/arptables
